web/ui/dashboard/src/app/private/pages/project/events/event-delivery-details/event-delivery-details.component.html

```diff
@@ -120,3 +123,9 @@ <h3 class="font-medium">{{ eventDelsDetails?.app_metadata?.name }}</h3>
 		</div>
 	</div>
 </convoy-card>
+
+<convoy-modal position="full" title="event payload" *ngIf="showPayload" (closeModal)="showPayload = false">
+    <div class="py-40px" modalBody>
+        <prism language="json" [code]="getCodeSnippetString('event_delivery')"></prism>
+    </div>
+</convoy-modal>
```

Comment:
I think this makes sense, but I'm not sure it works in this scenario. Instead of showing this modal, give the prism component a fixed height and auto scroll.

You might want to style the scroll bar such that it looks a little blended along with the design.

Reply:
We can review this implementation together.